Data_and_Model_Iteration.md

```diff
@@ -0,0 +1,22 @@
+## Capstone Assignment: Data and Model Iteration
+
+### Capstone: Samsung OCT 1
+#### Group Members:
+    1. Asieh Harati
+    2. Troy Jennings
+    3. Wolfgang Black
+
+## Questions     
+1. Perform feature engineering 
+    - For Neural Nets a large amount of traditional feature engineering is largely unnecessary. The benefit of the myriad of non-lienar connections allow the NN to learn 'features' and deprioritize the less useful features. Its one of the strengths of NN, but typically is the reason why larger datasets are necessary. For our particular use case, if you squint you could consider the image augmentation used in the simclr model to be like feature engineering. This augmentation is found in $get_augmenter$ function  (https://github.com/jenningst/fourthbrain-capstone/blob/main/simclr_test/simclr%20model%20V2.ipynb - cell 6) and can be seen to affect image crop, view, brightness, and color.
```

Comment:
Its > It's (sorry for being pedantic :D)
After you approve my PR, you can change 
cell 6 
to Image augmentations